### ISSUE TITLE
Document caveats about working with pm.sendRequest response headers

### ISSUE THREAD

**Issue by numaanashraf** [postmanlabs/postman-docs#1364]
User: I'm having real issues drilling down into a pm.sendRequest response header in a pre-request script. Syntax like res.header[x].value barfs in postman (the value is there, though), while the same JSON object in the browser console is fine. Known issue? #testing

Postman: The response you get in the callback is not a plain javascript object. It is a postman-collection response instance. You can find the API reference here https://t.co/rxLc4yFlo8

The documentation for send request https://t.co/BjbaqrVA1i has more details.

You might want to try this out
```
pm.sendRequest('http://postman-echo.com', function (err, res) {
  console.log(res.headers.toObject());
});
```

It doesn’t communicate the caveat with the response type. But it points to the link for the right response type.

**Comment by ArifPMan** [comment]
Validated in Prod

**Comment by LordDelacroix** [comment]
Did this ever get fixed?  I'm using 7.0.4 and try as I might - I am unable to get ANY of the response headers even though they ARE present.
